clients/src/main/java/org/apache/kafka/common/requests/PartitionState.java

```diff
@@ -42,8 +40,8 @@ public String toString() {
         return "PartitionState(controllerEpoch=" + controllerEpoch +
                 ", leader=" + leader +
                 ", leaderEpoch=" + leaderEpoch +
-                ", isr=" + Arrays.toString(isr.toArray()) +
+                ", isr=" + isr +
```

Comment:
I did it here and the other similar places that were touched by this PR. However, I think it's borderline if this is worth it. The default `toString` of `List` is `[1, 2, 3]` instead of `1, 2, 3`.